.gitignore

```diff
@@ -92,6 +92,6 @@ nuiEditorAutosave.json
 # Ignore stuff we extract for use with Terasology
 /natives/
 config/metrics/
-
+configs/engine/rendering.cfg
```

Comment:
We have a similar change at #3227 and this is not related to javadoc.
Can we exclude it here and have a separate PR for excluding files from the FlexibleConfig?